components/shell/bash/Makefile

```diff
@@ -124,16 +122,12 @@ COMPONENT_TEST_MASTER		= \
 # avoid permission issues for other users running these tests
 COMPONENT_POST_TEST_ACTION =	rm -f /tmp/xx
 
-build: $(BUILD_64)
-
 install: $(INSTALL_64)
```

Comment:
Move it to COMPONENT_POST_INSTALL_ACTION ?

Reply:
Changed `COMPONENT_POST_TEST_ACTION` to `COMPONENT_POST_INSTALL_ACTION`.

Reply:
Wait, I haven't meant it.
I've meant that lines `(cd $(PROTOUSRSHAREDIR)/locale ; cp -Rp "en@boldquot" en ;...` could be moved to COMPONENT_POST_INSTALL_ACTION, and so we don't need `install: $(INSTALL_64)...` rule